tests/components/airgradient/test_sensor.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for the AirGradient sensor platform."""
+
+from datetime import timedelta
+from unittest.mock import AsyncMock
+
+from airgradient import AirGradientError
+from freezegun.api import FrozenDateTimeFactory
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_all_entities(
```

Comment:
I would tie this test name to a specific model as they will ship different models in the future with different sensor configurations.